hoprd/hoprd/src/cli.rs

```diff
@@ -151,14 +151,6 @@ pub struct CliArgs {
     )]
     pub password: Option<String>,
 
-    #[arg(
-        long = "disableUnrealizedBalanceCheck",
-        env = "HOPRD_DISABLE_UNREALIZED_BALANCE_CHECK",
-        help = "Disables checking of unrealized balance before validating unacknowledged tickets.",
-        action = ArgAction::Count
-    )]
-    pub no_check_unrealized_balance: u8,
```

Comment:
Why was this removed? The PR does not state that.

Reply:
The check is not there anymore for a long time, so this option was dangling there doing nothing.
We kept it in 2.2 not to break the CLI args, but in 3.0 there's no reason to keep it.

I stumbled upon it when going through the ticket validation code in this PR.